babel.config.js

```diff
@@ -1,12 +1,26 @@
 // for ESM don't transpile modules
+
+const envConfig = {
+  ignoreBrowserslistConfig: true,
+  modules: 'commonjs',
+  targets: { node: true }
+};
+
+if (process.env.ESM) {
+  envConfig.modules = false
+}
+
+if (process.env.CDN) {
+  envConfig.modules = 'umd'
+  envConfig.targets = null
+  envConfig.ignoreBrowserslistConfig = false
+}
```

Comment:
I'd rather this used an `if/elseif/else` chain and instead of overwriting the initial properties of `envConfig` it set variables that were then used to create the `envConfig` object afterwards, but :man_shrugging:

Reply:
I'm so averse to else it's problematic, haha! will take note for further iterations